generators/client/templates/react/src/main/webapp/app/reducers/index.ts

```diff
@@ -5,12 +5,14 @@ import { loadingBarReducer as loadingBar } from 'react-redux-loading-bar';
 import locale from './locale';
 <%_ } _%>
 import layout from './layout';
+import account from './account';
```

Comment:
I think the settings pages are still using functions from `authentication.ts` file. we should change them to refer this

Reply:
you want merge account.ts and autentication.ts in one file authentication.ts ?

Reply:
No account related should be in account and we should remove it from authentication